app/src/views/disk-view.js

```diff
@@ -46,7 +47,9 @@ export default class DiskView {
   }
 
   _handleChanges(changes) {
-    if (changes.hasOwnProperty('disk') && changes.disk.hasOwnProperty('level') || changes.currentGame === "disk") {
+    if (changes.currentGame === GAMES.HANDSHAKE || // Reset on startup
```

Comment:
I believe the constructor is called before serial is initialized. To fix this we'd first have to refactor the initialization sequence.